src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -205,6 +206,24 @@ protected set
             }
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool TransformedByLayout
```

Comment:
Please leave a description comment in the case of hidden API.

Reply:
done. thank you